cli/central-client/src/main/java/org/ballerinalang/central/client/CentralAPIClient.java

```diff
@@ -143,14 +154,15 @@ public class CentralAPIClient {
     private static final MediaType JSON = MediaType.parse("application/json; charset=utf-8");
     private static final MediaType JSON_CONTENT_TYPE = MediaType.parse("application/json");
 
-    // System property name for enabling central verbose
-    public static final String SYS_PROP_CENTRAL_VERBOSE_ENABLED = "CENTRAL_VERBOSE_ENABLED";
     private static final int DEFAULT_CONNECT_TIMEOUT = 60;
     private static final int DEFAULT_READ_TIMEOUT = 60;
     private static final int DEFAULT_WRITE_TIMEOUT = 60;
     private static final int DEFAULT_CALL_TIMEOUT = 0;
     private static final int MAX_RETRY = 1;
     public static final String CONNECTION_RESET = "Connection reset";
+    private static final String ENV_CENTRAL_VERBOSE_ENABLED = "CENTRAL_VERBOSE_ENABLED";
+    private static final String ENV_TRUSTSTORE_PATH = "BALLERINA_TRUSTSTORE_PATH";
```

Comment:
Truststore is a java specific term. We need to find a generic term.

Reply:
https://central.ballerina.io/ballerina/http/2.12.3#ClientSecureSocket

Here we allow a truststore or a single cert.